assets/scss/components/_hscroll.scss

```diff
@@ -59,15 +59,29 @@ Class                       | Description
 	}
 }
 
+.hscrollContainer, .hscroll {
+	margin-bottom: -#{$space-and-half};
+}
+
+.hscrollContainer {
+	overflow: hidden;
+}
+
 .hscroll {
 	-webkit-overflow-scrolling: touch;
 	-ms-overflow-style: -ms-autohiding-scrollbar;
 	overflow-x: scroll;
+	-ms-overflow-style: auto
+}
+
+.hscroll::-webkit-scrollbar {
+	display: none !important;
 }
 
 .hscroll-content {
 	@extend %inlineblockList;
 	box-sizing: content-box;
+	margin-bottom: $space-and-half !important; // !important to kill styles coming from `sassquatch.css`, being imported by `swarm-sasstools`
```

Comment:
Ok, I see - this `!important` is required in docs to override sq2 defaults (because I used sq2 to do the docs site layout). I'll file a ticket to compile a version of sq2 for docs that does not include component classes.